django_restql/mixins.py

```diff
@@ -92,50 +110,48 @@ def fields(self):
         )
 
         if is_top_retrieve_request or is_top_list_request:
-            query_str = self.get_query_str(request)
-            parser = Parser(query_str)
-            try:
-                fields_query = parser.get_parsed()
-            except SyntaxError as e:
-                msg = (
-                    "QueryFormatError: " + 
-                    e.msg + " on " + 
-                    e.text
-                )
-                raise ValidationError(msg) from None           
+            if self.query is not None:
+                # The field_query is specified as kwarg
+                pass
+            else:
+                # Get field_query from the request
+                self.query = self.get_parsed_query_from_req(request)
```

Comment:
Incredibly minor (so feel free to ignore this), but this is functionally equivalent, and gets rid of an empty if case.
```python
            if self.query is None:
                self.query = self.get_parsed_query_from_req(request)
```

Reply:
Yeah you are right, let me change it.